x-pack/plugins/snapshot_restore/public/application/components/index.ts

```diff
@@ -11,7 +11,7 @@ export { RepositoryDeleteProvider } from './repository_delete_provider';
 export { RepositoryForm } from './repository_form';
 export { RepositoryVerificationBadge } from './repository_verification_badge';
 export { RepositoryTypeLogo } from './repository_type_logo';
-export { SectionLoading } from './section_loading';
+export { SectionLoading, InlineLoading, PageLoading } from './loading';
```

Comment:
I love the addition of these components. In the future we can explore moving them to `es_ui_shared` and using them broadly across our apps.